packages/teams-js/src/internal/telemetry.ts

```diff
@@ -290,6 +290,7 @@ export const enum ApiName {
   Settings_Save_Success = 'settings.save.success',
   Settings_SetSettings = 'settings.setSettings',
   Settings_SetValidityState = 'settings.setValidityState',
+  Sharing_History_GetContent = 'sharing.getContent',
```

Comment:
>sharing.getContent [](http://example.com/codeflow?start=32&length=18)

Should this be `sharing.history.getContent` ?

Reply:
yes updated